Holo-annie-jam/Managers/RhythmManager/Beatmap.cs

```diff
@@ -167,7 +167,17 @@ public Beatmap Build() {
 /// <summary>
 /// Represents a single event in a beatmap
 /// </summary>
-public class RhythmEvent {
+public class RhythmEvent : IComparable {
+
+    public int CompareTo(object obj) {
+        if (obj == null) return 1;
+
+        RhythmEvent otherRhythmEvent = obj as RhythmEvent;
+        if (otherRhythmEvent != null)
+            return (int)otherRhythmEvent.Tick - (int)this.Tick;
+        else
+            throw new ArgumentException("Object is not a Temperature");
```

Comment:
is not a what

Reply:
oh whoops LOL I stole this and forgot to rename the exception